### DISCUSSION TITLE
Doesn't run on Android 14 pixel 6,it keeps on crashing

### DISCUSSION THREAD

**Discussion by Taevoloev** [LSPosed/LSPosed#2499] [Q&A]
Have anyone successfully run lsposed on Android 14

**Comment by yujincheng08** [comment]
![image](https://user-images.githubusercontent.com/5022927/232406199-edc56d5d-788c-4a0e-a803-2b691e8b5171.png)
![image](https://user-images.githubusercontent.com/5022927/232406186-678fcaaa-4f2b-4a1f-b9c2-9db73e17485d.png)

**Reply by Taevoloev** [reply]
pixel 6 Android14 beta1 打不开管理器，直接闪退

**Reply by yujincheng08** [reply]
我这个就是 beta 1

**Reply by Taevoloev** [reply]
pixel6,7都不行![Screenshot_20230423-115047.png](https://user-images.githubusercontent.com/124876450/233818865-417e92b7-5dc8-492b-a3c5-30bdc9740eec.png)
